.travis.yml

```diff
@@ -36,6 +36,7 @@ matrix:
       env: SLS_IGNORE_WARNING=*
 sudo: false
 install:
+  - "[ $TRAVIS_OS_NAME = windows ] || travis_retry npm install -g npm"
```

Comment:
npm i -g npm doesn't work on windows travis builders bc apparently the user doesn't have enough permission, and it's not necessary bc the windows build has a new enough node/npm that it doesn't need to be upgraded.

Reply:
As we clarified on Slack, it's about invalid listing of `zlib` in package.json. it's old module that doesn't compile on new node.js versions (recent ones totally ignore it), and crashes v6.

`zlib` is a Node.js native module since v0.12.

Removing it from `package.json` will fix it